internal/plan/plan.go

```diff
@@ -172,21 +205,21 @@ func CombineLayers(layers ...*Layer) (*Layer, error) {
 // An error is returned when a provided service name does not exist, or there
 // is an order cycle involving the provided service or its dependencies.
 func (p *Plan) StartOrder(names []string) ([]string, error) {
-	return order(p.Services, names, false)
+	return order(p.Services, names, false, true)
 }
 
 // StopOrder returns the required services that must be stopped for the named
 // services to be properly stopped, in the order that they must be stopped.
 // An error is returned when a provided service name does not exist, or there
 // is an order cycle involving the provided service or its dependencies.
-func (p *Plan) StopOrder(names []string) ([]string, error) {
-	return order(p.Services, names, true)
+func (p *Plan) StopOrder(names []string, includeDependants bool) ([]string, error) {
```

Comment:
This needs to be looked into per note on #58: I suggest leaving the original logic in the package alone, and just having that sorting function that applies sorting from a list into a different subset list.